### ISSUE TITLE
alpha@6 Issues with Newly Created Connectors

### ISSUE THREAD

**Issue by rogueSF** [strapi/strapi#267]
I am having some issues with model creation using newly created connectors in alpha@6.  I pulled the latest updates yesterday morning which fixed my inability to connect to my local mongo instance, however, whenever a new model is created using my new connector, I receieve the following error

`
ERROR (5902 on ********): Impossible to register the `Report` model.
ERROR (OverwriteModelError/5902 on *****): Cannot overwrite `Report` model once compiled.
    OverwriteModelError: Cannot overwrite `Report` model once compiled.
        at OverwriteModelError (/Users/smstewa5/Code/Strapi/strapi/packages/strapi-mongoose/node_modules/mongoose/lib/error/overwriteModel.js:18:11)
        at Mongoose.model (/Users/smstewa5/Code/Strapi/strapi/packages/strapi-mongoose/node_modules/mongoose/lib/index.js:405:13)
        at _.forEach (/Users/smstewa5/Code/Strapi/strapi/packages/strapi-mongoose/lib/index.js:142:58)
        at /Users/smstewa5/Code/Strapi/strapi/packages/strapi-mongoose/node_modules/lodash/lodash.js:4944:15
        at baseForOwn (/Users/smstewa5/Code/Strapi/strapi/packages/strapi-mongoose/node_modules/lodash/lodash.js:3001:24)
        at /Users/smstewa5/Code/Strapi/strapi/packages/strapi-mongoose/node_modules/lodash/lodash.js:4913:18
        at Function.forEach (/Users/smstewa5/Code/Strapi/strapi/packages/strapi-mongoose/node_modules/lodash/lodash.js:9359:14)
        at _.after (/Users/smstewa5/Code/Strapi/strapi/packages/strapi-mongoose/lib/index.js:84:15)
        at /Users/smstewa5/Code/Strapi/strapi/packages/strapi-mongoose/node_modules/lodash/lodash.js:10014:23
        at _.after (/Users/smstewa5/Code/Strapi/strapi/packages/strapi-mongoose/lib/index.js:204:15)
        at /Users/smstewa5/Code/Strapi/strapi/packages/strapi-mongoose/node_modules/lodash/lodash.js:10014:23
        at _.forEach (/Users/smstewa5/Code/Strapi/strapi/packages/strapi-mongoose/lib/index.js:293:15)
        at /Users/smstewa5/Code/Strapi/strapi/packages/strapi-mongoose/node_modules/lodash/lodash.js:4944:15
        at baseForOwn (/Users/smstewa5/Code/Strapi/strapi/packages/strapi-mongoose/node_modules/lodash/lodash.js:3001:24)
        at /Users/smstewa5/Code/Strapi/strapi/packages/strapi-mongoose/node_modules/lodash/lodash.js:4913:18
        at Function.forEach (/Users/smstewa5/Code/Strapi/strapi/packages/strapi-mongoose/node_modules/lodash/lodash.js:9359:14)
`

A bit of research showed the error is due to an already created collection but this isnt the case because I'm using a new db.  Changing the collection name has no effect either, I get the same end result.  

I knew this had to be a bug when instead of creating new connectors, I instead took the faulty connectors configuration and applied it exactly the same to the default connector and everything 
works perfectly fine without issue.  

TL;DR:

Newly created connectors appear to have issues with model creation.  The settings used which dont work with a new custom connector work just fine if applied to the default connector

**Connector Type**: Mongo/Mongoose
**Node.js version**: 8.6.0
**npm version**: 5.4.2
**Strapi version**: alpha@6
**Operating system**: os x 10.11

**Comment by Aurelsicoko** [comment]
Thanks for reporting this issue. We fixed it this morning!

**Comment by rogueSF** [comment]
Nice work!
